builder/util/util.go

```diff
@@ -0,0 +1,83 @@
+package util
```

Comment:
Ick. Can you put a note that this should go back into that package if that problem is solved?